suite-native/atoms/src/SelectableItem.tsx

```diff
@@ -4,15 +4,17 @@ import { ReactNode } from 'react';
 import { useTranslate } from '@suite-native/intl';
 import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
 
-import { Card } from './Card/Card';
 import { VStack } from './Stack';
 import { Box } from './Box';
 import { Badge } from './Badge';
 import { Text } from './Text';
 import { Radio } from './Radio';
 
 const cardStyle = prepareNativeStyle((utils, { isSelected }: { isSelected: boolean }) => ({
+    backgroundColor: utils.colors.backgroundSurfaceElevation1,
```

Comment:
Aren't we duplicating styles from `Card` component now? What about just importing the `cardStyle` object from there?

Reply:
Yes, because it has odd ontouch UI on android due to TouchableOpacity. Importing cardstyle is good idea

Reply:
But here I dont want to make it ready for AlertBox and I want to change the style a bit. Updating cardStyle just for  that feels suboptimal. I will keep it this way.